### ISSUE TITLE
Make `auth` contract depend on the fresh `worker` artifact

### ISSUE THREAD

**Issue by itegulov** [near/email-auth#7]
Currently, we rely on the developer to put `worker.wasm` in the `auth/src` folder which is an error-prone pattern. A better way would be to depend on `../../worker/target/.../worker.wasm`, but even then the developer could forget to rebuild the worker contract. I don't think there is an easy way to achieve what we want, but I have [added](https://github.com/near/workspaces-rs/blob/main/workspaces/src/cargo/mod.rs) a "please compile this project and give me its WASM artifact"-function to workspaces which could come useful.

**Comment by volovyks** [comment]
I just copied the current version of it to src folder for now, or yes, we can change the path. What about modifiyng the build scriot? 1. Build contract 1, 2. Copy, 3. Build contract 2. I do not think that we will nedd to update controller contract often, let's not invest a lot of time to it.

**Comment by itegulov** [comment]
Yeah this might not be a big deal right now, but I had to update the worker contract a few times while working on #2. One other observation I made is that we don't have an upgrade mechanism for worker contracts. Every time I changed `worker.wasm` I had to use a fresh testnet account that can create fresh email subaccounts which was pretty annoying.

In any case, I agree that this is very low priority. Just though I would post my thoughts on how to improve this in the future.
